client/src/main/java/io/quarkus/vault/client/common/VaultBinaryResult.java

```diff
@@ -0,0 +1,15 @@
+package io.quarkus.vault.client.common;
+
+public class VaultBinaryResult implements VaultResult {
```

Comment:
FYI this is never used

Reply:
Correct and I can remove it but there are Vault responses that only return "raw" data (no leasing or even JSON). Previously, we were something like this for the old PKI interface but I realized it was possibly (and easier) to always use the JSON responses for PKI. 

That being said, I think this should remain for the time when an API that returns raw data is implemented or when an end user needs the capability when accessing a custom plugin API.

Reply:
ok. so may be it is worth a comment.